scripts/createC2DExamples-mac.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Create markdown file
+cp test/integration/C2DExamples.test.ts C2DExamples.md
```

Comment:
why does this file exist twice, indicating that macOS needs different build process but then both files are the same? There should be no multiple different build processes but rather one script dealing with OS differences inside of it